clientv3/integration/watch_test.go

```diff
@@ -396,3 +396,80 @@ func TestWatchCompactRevision(t *testing.T) {
 		t.Fatalf("expected closed channel, but got %v", wresp)
 	}
 }
+
+func TestWatchWithProgressNotify(t *testing.T) {
+	defer testutil.AfterTest(t)
+
+	tests := []struct {
+		currentRevision int64
+		events          []*storagepb.Event
+		compactRevision int64
+		canceled        bool
+	}{
+		{1, []*storagepb.Event{}, 0, false},
+		{1, []*storagepb.Event{}, 0, false},
+		{1, []*storagepb.Event{}, 0, false},
```

Comment:
are these progress notifications in response to the PUTs? If so, shouldn't this be testing that the revisions are increasing?